pkg/routing/utils.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func participantKey(roomName livekit.RoomName, identity livekit.ParticipantIdentity) string {
-	return roomName + "|" + identity
+	return string(roomName) + "|" + identity
```

Comment:
`string(roomName) + "|" + string(identity)` (are these needed?)

Reply:
Will be needed once identity becomes a type definition too.